[BUG] Undeclared error in: "tensorflow/contrib/memory_stats/kernels/memory_stats_ops.cc"

### System information
- **Linux Ubuntu 17.10**:
- **TensorFlow installed from source**:
- **TensorFlow version 1.4.1**:
- **Python version 3.6**: 
- **Bazel version (0.8.1)**:
- **GCC/Compiler version (7.2)**:

### Describe the problem
undeclared error raised in "tensorflow/contrib/memory_stats/kernels/memory_stats_ops.cc" when build v1.4.1 with jemalloc, OpenCL

### Source code / logs
ERROR: .../tensorflow/tensorflow/contrib/memory_stats/BUILD:17:1: C++ compilation of rule '//tensorflow/contrib/memory_stats:python/ops/_memory_stats_ops.so' failed (Exit 1)
tensorflow/contrib/memory_stats/kernels/memory_stats_ops.cc:64:5: error: unknown type name '**MaxBytesInUseOp**'; did you mean 'BytesInUseOp'?
    MaxBytesInUseOp);
    ^~~~~~~~~~~~~~~
    BytesInUseOp
./tensorflow/core/framework/op_kernel.h:1209:68: note: expanded from macro 'REGISTER_KERNEL_BUILDER'
  REGISTER_KERNEL_BUILDER_UNIQ_HELPER(__COUNTER__, kernel_builder, __VA_ARGS__)
                                                                   ^
./tensorflow/core/framework/op_kernel.h:1212:53: note: expanded from macro 'REGISTER_KERNEL_BUILDER_UNIQ_HELPER'
  REGISTER_KERNEL_BUILDER_UNIQ(ctr, kernel_builder, __VA_ARGS__)
                                                    ^
./tensorflow/core/framework/op_kernel.h:1225:24: note: expanded from macro 'REGISTER_KERNEL_BUILDER_UNIQ'
            return new __VA_ARGS__(context);                          \
                       ^
tensorflow/contrib/memory_stats/kernels/memory_stats_ops.cc:44:7: note: 'BytesInUseOp' declared here
class BytesInUseOp : public MemoryStatsOp {
      ^
1 error generated.

--- a/tensorflow/contrib/memory_stats/kernels/memory_stats_ops.cc
+++ b/tensorflow/contrib/memory_stats/kernels/memory_stats_ops.cc
@@ -59,7 +59,7 @@ REGISTER_KERNEL_BUILDER(Name("BytesInUse").Device(DEVICE_GPU).HostMemory("out"),
 
 #ifdef TENSORFLOW_USE_SYCL
 REGISTER_KERNEL_BUILDER(
-    Name("BytesInUse").Device(DEVICE_SYCL).HostMemory("out"), MaxBytesInUseOp);
+    Name("BytesInUse").Device(DEVICE_SYCL).HostMemory("out"), BytesInUseOp);
 #endif  // TENSORFLOW_USE_SYCL
 
 // Op that measures the total memory (in bytes) of a device.

